src/redux/selectors/index.js

```diff
@@ -470,3 +470,110 @@ export const accordionsContainerDataSelector = createSelector(
     return subTypeData;
   },
 );
+
+export const filterTriggerDateRangeSelector = createSelector(
```

Comment:
could we use this same selector as input for `timelineIntervalsSelector` ?
(replacing `timelineItemsInRangeSelector` ?)

...if so, this should be above it.